src/framework/preferences/html/PrefsEditorTemplate-contrast.html

```diff
@@ -7,4 +8,4 @@
         <span class="fl-hidden-accessible"></span>
         <span class="fl-crossout"></span>
     </label>
-</div>
+</div>
```

Comment:
Could you please make sure that all the files have an empty new line at the end. This will remove the complaint from GitHub. There were some editors, at least historically, that would have issues with files that didn't end in an empty new line.